charts/opentelemetry-operator/values.yaml

```diff
@@ -42,7 +42,7 @@ manager:
     tag: ""
   collectorImage:
     repository: ""
-    tag: 0.122.1
+    tag: 0.123.0
```

Comment:
```suggestion
    tag: 0.123.1
```

Reply:
What repository does the operator use by default? in v0.123.0 release we stopped publishing the collector to Dockerhub.

Reply:
Operator uses ghcr for everything. I guess the e2e tests in this repo still use dockerhub? Let me update that.